src/battleship/org/Battlefield.java

```diff
@@ -1,21 +1,28 @@
-package engine;
+package battleship.org;
 
-import java.util.Arrays;
+import battleship.org.equipment.*;
+
+import java.util.*;
 
 
 public class Battlefield {
     private final Designations[][] battleField = new Designations[Constant.FIELD_SIZE][Constant.FIELD_SIZE];
```

Comment:
Instead of `Designations[][] battleField` create custom class Cell (or some other name) that will contain current state and all needed information
Cell[][] battleField